source/scripts/updateFields.js

```diff
@@ -0,0 +1,44 @@
+const PouchDB = require('pouchdb');
+PouchDB.plugin(require('pouchdb-find'));
+const db = new PouchDB('db');
+
+/**
+ *
+ * @param {res} res The response by which to send back the results of
+ * the operation.
+ * @param {*} req The request that contains the data to update the
+ * field.
+ * 
```

Comment:
Codacy has a fix for the issue: [Trailing spaces not allowed.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7417876)
```suggestion
 *
```